ow_power_system/include/power_system_node.h

```diff
@@ -17,20 +21,28 @@ class PowerSystemNode
 
 private:
   bool loadSystemConfig();
-  std::vector<std::map<MessageId, Datum<double>>> loadPowerProfile(const std::string& filename);
+  PrognoserVector loadPowerProfile(const std::string& filename);
   bool loadFaultPowerProfiles();
   bool initPrognoser();
   bool initTopics();
   void jointStatesCb(const sensor_msgs::JointStateConstPtr& msg);
   double generateTemperatureEstimate();
   double generateVoltageEstimate();
-  void injectFault(const std::string& power_fault_name, bool& fault_activated,
-                   const std::vector<std::map<PCOE::MessageId, PCOE::Datum<double>>>& sequence, size_t& sequence_index,
-                   double& power, double& voltage, double& temperature);
+  void injectFault(const std::string& power_fault_name,
+                   bool& fault_activated,
+                   const PrognoserVector& sequence,
+                   size_t& sequence_index,
+                   double& power,
+                   double& voltage,
+                   double& temperature);
   void injectFaults(double& power, double& temperature, double& voltage);
-  std::map<PCOE::MessageId, PCOE::Datum<double>> composePrognoserData(double power, double voltage, double temperature);
+  PrognoserMap composePrognoserData(double power,
+                                    double voltage,
+                                    double temperature);
   void parseEoD_Event(const ProgEvent& eod_event,
-                      std_msgs::Float64& soc_msg, std_msgs::Int16& rul_msg, std_msgs::Float64& battery_temperature_msg);
+                      std_msgs::Float64& soc_msg,
+                      std_msgs::Int16& rul_msg,
+                      std_msgs::Float64& battery_temperature_msg);
   void powerCb(double electrical_power);
```

Comment:
`powerCb` contains the "Cb" abbreviation, but is never used as a callback function. And I think it could use a more descriptive name.

Reply:
Pre-existing code, but good catch.  I've renamed it to runPrognoser().